src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -291,7 +291,7 @@ USER_REPORTING_METADATA_UPDATE_FREQUENCY = {{ localsettings.USER_REPORTING_METAD
 SMS_QUEUE_ENABLED = {{localsettings.get('SMS_QUEUE_ENABLED', False)}}
 REMINDERS_QUEUE_ENABLED = {{localsettings.get('REMINDERS_QUEUE_ENABLED', False)}}
 
-PILLOW_RETRY_QUEUE_ENABLED = {{ localsettings.get('PILLOW_RETRY_QUEUE_ENABLED', False) }}
+PILLOW_RETRY_QUEUE_ENABLED = True
```

Comment:
Will there be a followup to remove this after https://github.com/dimagi/commcare-hq/pull/25126/ ?

Reply:
yes